utils/staking-miner/src/main.rs

```diff
@@ -249,6 +279,65 @@ enum Command {
 	EmergencySolution,
 }
 
+#[derive(Debug, Clone, StructOpt)]
+enum Solvers {
+	SeqPhragmen {
+		#[structopt(long, default_value = "10")]
+		iterations: u32,
+	},
+	PhragMMS {
+		#[structopt(long, default_value = "10")]
+		iterations: u32,
+	},
+}
+
+/// Mine a solution with the given `solver`.
+fn mine_with<T>(
+	solver: &Solvers,
+	ext: &mut Ext,
+) -> Result<(EPM::RawSolution<EPM::SolutionOf<T>>, u32), Error<T>>
+where
+	T: EPM::Config,
+	T::Solver: NposSolver<Error = sp_npos_elections::Error>,
+{
+	use frame_election_provider_support::{PhragMMS, SequentialPhragmen};
+
+	match solver {
+		Solvers::SeqPhragmen { iterations } => {
+			BalanceIterations::set(*iterations);
+			mine_unchecked::<
+				T,
+				SequentialPhragmen<
+					<T as frame_system::Config>::AccountId,
+					sp_runtime::Perbill,
+					Balancing,
+				>,
+			>(ext, false)
+		},
+		Solvers::PhragMMS { iterations } => {
+			BalanceIterations::set(*iterations);
+			mine_unchecked::<
+				T,
+				PhragMMS<<T as frame_system::Config>::AccountId, sp_runtime::Perbill, Balancing>,
+			>(ext, false)
+		},
+	}
+}
+
+frame_support::parameter_types! {
+	/// Number of balancing iterations for a solution algorithm. Set based on the [`Solvers`] CLI
+	/// config.
+	pub static BalanceIterations: u32 = 10;
+}
+
+/// Balancing configuration for for the solution algorithm. See [`Solvers`] for config options.
+struct Balancing;
+impl Get<Option<(usize, ExtendedBalance)>> for Balancing {
+	fn get() -> Option<(usize, ExtendedBalance)> {
+		Some((BalanceIterations::get() as usize, 0))
+	}
+}
```

Comment:
```suggestion
	pub static BalanceIterations: u32 = 10;
	pub static Balacing: (usize, ExtendedBalance) = (BalanceIterations::get(), 0);
}
```

not sure, but this might work as well.